src/consts.ts

```diff
@@ -23,15 +23,15 @@ export const U32Opts = { bitLength: 32, isLe: true };
 export const PayeeStatus = [
   {
     key: 'Staked',
-    name: 'Back to Staking',
+    name: 'consts.staked',
```

Comment:
This `name` key needs to be changed, and `PayeeStatus` itself probs should be moved.

Reply:
please check the updates